Assets/Scripts/Editor/BeatMappingWindow.cs

```diff
@@ -0,0 +1,198 @@
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+using UnityEditor.UIElements;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+using ProefExamen.Audio.WaveFormDrawer;
+using ProefExamen.Audio.SpectrumDrawer;
+using ProefExamen.Audio.TimeStamping;
+
+namespace ProefExamen.Editor.MappingWindow
```

Comment:
Same for this namespace. I would create a folder named `BeatMapping` and change the namespace to `ProefExamen.Editor.BeatMapping`